terraform-modules/k8s-master/variables.tf

```diff
@@ -38,23 +60,23 @@ variable private_ipv4_cidr_block {
 }
 
 variable enable_private_nodes {
-  type = bool
+  type    = bool
   default = true
 }
 
 variable enable_private_endpoint {
-  type = bool
+  type    = bool
   default = false
 }
 
 variable use_new_stackdriver_apis {
-  type = bool
-  default = true
+  type        = bool
+  default     = true
```

Comment:
Is there a terraform linting tool you used for this?

Reply:
I installed terraform lint and syntax support in Atom, and it reformats when I save. Seems to produce similar results to `terraform fmt`